td.vue/package.json

```diff
@@ -20,9 +24,7 @@
     "lint": "vue-cli-service lint",
     "codecov": "codecov"
   },
-  "resolutions": {
-    "ansi-regex": "^5.0.1"
-  },
+  "main": "background.js",
```

Comment:
if we rename away from the default `background.js` then this line may need to be updated

Reply:
Removed this line! Thank you for your suggestion.